addons/base_automation/models/base_automation.py

```diff
@@ -291,7 +291,12 @@ def make_onchange(action_rule_id):
             def base_automation_onchange(self):
                 action_rule = self.env['base.automation'].browse(action_rule_id)
                 result = {}
-                server_action = action_rule.action_server_id.with_context(active_model=self._name, onchange_self=self)
+                server_action = action_rule.action_server_id.with_context(
+                    active_model=self._name,
+                    onchange_self=self,
+                    active_id=None,
+                    active_ids=None
```

Comment:
Shouldn't `active_ids` map to a list instead?
```suggestion
                    active_id=None,
                    active_ids=[],
```
From version 13.0 on, the record id should be the origin record's id:
```suggestion
                    active_id=self._origin.id,
                    active_ids=self._origin.ids,
```

Reply:
active_ids is fixed.
I'll ``self._origin.id(s)`` during forrward-porting